Header 								header
string                              model
DetectedPoint 						top_left
DetectedPoint 						top_right
DetectedPoint 						bottom_right
DetectedPoint 						bottom_left
